app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
```

Comment:
Code Quality Issue: The variable `masks_to_buy` is initialized but not used in the case when all friends are vaccinated and are wearing masks. Consider moving this initialization inside the `except NotWearingMaskError:` block.